docs/references/nextjs/trpc.mdx

```diff
@@ -94,11 +94,24 @@ export const createContext = async (
 export type Context = trpc.inferAsyncReturnType<typeof createContext>;
 ```
 
-## Accessing the context data in your application
+Then, add the Clerk context to every tRPC query sent to the server:
```

Comment:
```suggestion
Next step, add the Clerk context to every tRPC query sent to the server:
```

Reply:
I think the first way sounds better. 
We would write "Next step" if it was in another step, but in this case, it is not. It's just another instruction in the same step.

Reply:
I would drop the 'Next step' or 'Then' and just do 'Add the Clerk....'